acis_thermal_check/state_builder.py

```diff
@@ -0,0 +1,254 @@
+import os
+import Ska.DBI
+from Chandra.Time import DateTime
+from pprint import pformat
+import Chandra.cmd_states as cmd_states
+from acis_thermal_check.utils import globfile, mylog
```

Comment:
Using a global `mylog` still forces applications to use the ACIS logger which may not be desirable.  What about making `logger` be an `__init__` kwarg (`logger=None`):
```
if logger is None:
    # Make a logger but with no output
    logger = logging.getLogger('statebuilder-no-logger')
self.logger = logger
```